Maquetado/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <pre><h1 class="title">Título del libro<h1 class='gen'> (género)</h1></h1> </pre>
+  <br>
+  <br>
```

Comment:
Avoid ```br``` tags. You should manage the sapcing from the scss.

Reply:
Adding more info to this suggestion. The `br` tag intended use is to add line breaks to the contents of the same text.
Quoting the HTML specification:
> [br](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-br-element) elements must be used only for line breaks that are actually part of the content, as in poems or addresses.

The issue here is that you are trying to apply a layout styling with the line brakes generated by the br, which is semantically incorrect because there we are not separating the  "same content".

As Andres said, the spacing and visual styling should be addressed with css/scss